tests/extension/oneapi_property_list/property_list_prop_eq_op.cpp

```diff
@@ -15,8 +15,7 @@ namespace TEST_NAMESPACE {
 using namespace sycl_cts;
 
 template <typename T, typename U>
-constexpr void check_equal(util::logger &log, T prop1, U prop2,
-                           bool expected_equal) {
+void check_equal(util::logger &log, T prop1, U prop2, bool expected_equal) {
```

Comment:
Given that we have Catch2 now, wouldn't it be more readable to instead simply write something like

```c++
CHECK(prop_value_device_image_scope == prop_value_device_image_scope);
CHECK_FALSE(prop_value_device_image_scope != prop_value_device_image_scope);
```

?

Reply:
Done in https://github.com/KhronosGroup/SYCL-CTS/pull/291/commits/3c74460d5577cce2bb4e3f74e36de58e6e9b5c2d